### PR TITLE
Added --name parameter to allow setting the name of documented module

### LINKED ISSUE #98: Add '--name' paramter to replace 'main' in our tool
Name configuration based on project name rather thren use `main`

### PR DESCRIPTION
**Pull request by pikinier20**
closes #98

### CODE REVIEW

File: src/main/scala/dotty/dokka/Main.scala [hunk 1]
```diff
@@ -25,6 +25,8 @@ class RawArgs:
     @COption(name="--classpath", aliases = Array("--cp", "-c"), usage="Classpath to load depenecies from")
     private var classpath: String = System.getProperty("java.class.path")
 
+    @COption(name="--name", aliases = Array("-n"), usage="Name of documented project")
```
Comment (romanowski) [inline src/main/scala/dotty/dokka/Main.scala]: IMHO it should be mandatory

**Review by romanowski — CHANGES_REQUESTED** [review]

**Comment by romanowski** [comment]
I forgot to add to review but we also need to update documentation (yes, we have a documentation for cmd :) )

File: src/main/scala/dotty/dokka/Main.scala (hunk 1 above)
Comment (pikinier20) [reply src/main/scala/dotty/dokka/Main.scala]: Added `required` flag

**Comment by pikinier20** [comment]
Updated documentation, idk if "name of module in generated documentation" is proper term

**Review by romanowski — APPROVED** [review]

**Pull request merged**